cms/djangoapps/contentstore/views/course.py

```diff
@@ -679,7 +680,13 @@ def _format_library_for_view(library, request, migrated_to: Optional[NamedTuple]
         'org': library.display_org_with_default,
         'number': library.display_number_with_default,
         'can_edit': has_studio_write_access(request.user, library.location.library_key),
-        **(migrated_to._asdict() if migrated_to is not None else {}),
+        'is_migrated': migration is not None,
+        **({
+            'migrated_to_key': migration.target_key,
+            'migrated_to_title': migration.target_title,
+            'migrated_to_collection_key': migration.target_collection_slug,
+            'migrated_to_collection_title': migration.target_collection_title,
+        } if migration else {}),
```

Comment:
we're now using a general ModulestoreMigration dataclass object instead of a namedtuple that was made specifically for this API, so this just adds some marshalling.

Reply:
Nit: This is clever, but I feel like it would be easier to read if it was a conditional assignment to an `extra_kwargs` var above it.